scripts/download.js

```diff
@@ -3,8 +3,9 @@ var fs = require("fs");
 var url = require("url");
 
 function writeToFile(filename, response, callback) {
-  response.pipe(fs.createWriteStream(filename));
-  response.on("end", callback);
+  var writeStream = fs.createWriteStream(filename);
```

Comment:
```suggestion
  const writeStream = fs.createWriteStream(filename);
```

Reply:
since when does the `const` exist (which node version) and what's the minimum version of the node we support?

Reply:
Since the code already uses `var`, this is fine. We don't support deprecated Node versions, so I might run Eslint to update all of these later.